skops/card/_model_card.py

```diff
@@ -401,9 +394,39 @@ def save(self, path: str | Path) -> None:
                 model_plot=self._model_plot,
                 **template_sections,
             )
+        return card
+
+    def save(self, path: str | Path) -> None:
+        """Save the model card.
 
+        This method renders the model card in markdown format and then saves it
+        as the specified file.
+
+        Parameters
+        ----------
+        path: str, or Path
+            filepath to save your card.
+
+        Notes
+        -----
+        The keys in model card metadata can be seen `here
+        <https://huggingface.co/docs/hub/models-cards#model-card-metadata>`__.
+        """
+        card = self._generate_card()
         card.save(path)
```

Comment:
should this call `render`?

Reply:
You mean call `self.render` and then write the string directly to the file, instead of going through `card.save`?

Reply:
yeah that's a good question. I'm wondering if there _could_ be a difference between users calling save and saving the output of `render`, which wouldn't be ideal I think. WDYT?

Reply:
At the moment, both would amount to the same, but who knows what else could be added on the `modelcards` side in the future. Therefore, I would prefer users to use skops `Card.save`, which in turn should use `ModelCard.save`. But if a user wants to call `render` and save that, I guess we can't prevent them.